src/core/stdc/math.d

```diff
@@ -215,65 +215,152 @@ version( CRuntime_DigitalMars )
     }
   }
 }
-else version( CRuntime_Microsoft )
+else version( CRuntime_Microsoft ) // fully supported since MSVCRT 12 (VS 2013) only
 {
+  version( all ) // legacy stuff to be removed in the future
+  {
     enum
     {
-        ///
```

Comment:
This was actually intentional for these MS specific symbols. They haven't had these `///` comments before 2.067, so I thought it'd be better to leave these non-standard declarations uncommented and only decorate the standard ones while rebasing.
I can add them if you insist. :)